Main.cpp

```diff
@@ -202,6 +206,9 @@ namespace RTE {
 		long long drawTotalTime = 0;
 
 		while (!System::IsSetToQuit()) {
+
+			g_UInputMan.Update();
```

Comment:
This breaks input handling, because it's also updated in TimeForSimUpdate, which means it swallows inputs at high FPS.

Suggest this is just removed. It's unnecessary